torch/lib/c10d/ProcessGroup.hpp

```diff
@@ -257,7 +258,17 @@ class ProcessGroup : public torch::CustomClassHolder {
   virtual c10::intrusive_ptr<ProcessGroup::Work> reduce_scatter(
       std::vector<at::Tensor>& outputTensors,
       std::vector<std::vector<at::Tensor>>& inputTensors,
-      const ReduceScatterOptions& opts = ReduceScatterOptions()) = 0;
+      const ReduceScatterOptions& = ReduceScatterOptions()) = 0;
```

Comment:
nit: keep the same as before

Reply:
the linter issued a warning. I will overwrite that.